modules/statistics/RTPStatsCollector.js

```diff
@@ -453,6 +453,27 @@ StatsCollector.prototype.getNonNegativeStat = function(report, name) {
     return Math.max(0, value);
 };
 
+/**
+ * @param rtcPeerConnection
```

Comment:
Could you please describe this method in more details and add type declarations to JSDoc ?